spark-plugins/docs/DataCacher-sparkcompute.md

```diff
@@ -0,0 +1,27 @@
+# Data Cacher
```

Comment:
Missing copyright comment

Reply:
Do we add copyright comments to markdown files? I dont see the copyright comment in any of the other plugin docs

Reply:
I think we should have copyright in all files in the source repo

Reply:
ok, added copyright to this doc and the rest of the docs